lib/cli/lib/project_types.js

```diff
@@ -8,6 +8,8 @@ module.exports = {
   REACT_PROJECT: 'REACT_PROJECT',
   WEBPACK_REACT: 'WEBPACK_REACT',
   VUE: 'VUE',
+  WEBPACK_VUE: 'WEBPACK_VUE',
```

Comment:
`WEBPACK_VUE` and `BROWSERIFY_VUE` might refactor as SFO_VUE(Single file components) due to same logic.